mlflow/utils/rest_utils.py

```diff
@@ -215,6 +215,10 @@ def get_trace_info_endpoint(request_id):
     return f"{_REST_API_PATH_PREFIX}/mlflow/traces/{request_id}/info"
 
 
+def get_delete_tags_endpoint(request_id):
```

Comment:
Should this be `get_trace_tags_endpoint`?

Reply:
renamed!